### PR TITLE
Typo and consistency fix

### PR DESCRIPTION
**Pull request by Butek69412**

### CODE REVIEW

File: YAML/Map014.yaml
Comment (dazedanon) [inline YAML/Map014.yaml:1]: Why remove the quotes here if there is a speaker?

File: YAML/Map011.yaml
Comment (dazedanon) [inline YAML/Map011.yaml:1]: Why remove the quotes here if there is a speaker?

File: YAML/Map026.yaml [hunk 1]
```diff
@@ -432,7 +432,7 @@
       - !ruby/object:RPG::EventCommand {'i': !!int '0', 'c': !!int '401', 'p': ['【Aurora】']}
       - !ruby/object:RPG::EventCommand {'i': !!int '0', 'c': !!int '401', 'p': ['Hey...! Where do you think you''re touching!? Pervert! Total
 
-            lech!']}
+            leech!']}
```
Comment (dazedanon) [inline YAML/Map026.yaml:435]: This is correct. Lech is a derogatory term for someone perverted.

**Review by dazedanon — COMMENTED** [review]
I don't believe removing those quotes is correct. Since the speaker is talking, their sentences should be quoted. Otherwise lech is correct and shouldn't be leech.

**Comment by Butek69412** [comment]
There's a lot of inconsistencies with quotes, not all of the dialogue uses them, hence why i figured it was a gpt mistake. Besides that the game has a proper dialogue box with names, so i don't think it's needed.

File: YAML/Map026.yaml (hunk 1 above)
Comment (Butek69412) [reply YAML/Map026.yaml]: Fixed it

**Comment by dazedanon** [comment]
Hmm well I see your point. Author doesn't generally use Japanese quotes. However, let me remove them all with a mass replace instead. That way its consistent throughout the game. Pull the latest when you get the chance.

**Pull request closed without merging**